### DISCUSSION TITLE
Svelteのシンタックスハイライトの不具合

### DISCUSSION THREAD

**Discussion by shamokit** [increments/qiita-discussions#610] [要望・アイデア・不具合]
## 最初に
Svelteのシンタックスハイライトはかなり嬉しい機能追加でした！
ありがとうございます！

## 概要

以下ような書き方(`{...props}`)をすると赤色の波線が引かれてしまいます。

```svelte
<button use:action {...$trigger(id)}>
```

```svelte
<button {...props}>test</button>
```

### 例
https://qiita.com/shamokit/items/72c5cb7c62085f120673

**Comment by wataru86** [comment]
@shamokit
ご意見ありがとうございます🙏

Qiita のハイライトは Rouge を使用しており、こちらの構文を Rouge がエラーと判定しているため赤線が出てしまいます...
https://rouge.jneen.net/v4.2.0/svelte/PGJ1dHRvbiB1c2U6YWN0aW9uIHsuLi4kdHJpZ2dlcihpZCl9Pgo8YnV0dG9uIHsuLi5wcm9wc30-dGVzdDwvYnV0dG9uPg

Qiita 側で赤線を消すことは可能ですが、他のエラーのハイライトにも影響が出るため難しいと考えています。
Rouge 側の修正も含め、この問題を修正できないか検討いたします 🙇

**Reply by shamokit** [reply]
わかりました！
ありがとうございます🙇
